modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -3,50 +3,58 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can have multiple instances in the same environment. 
+In API Manager 2.x, instances of an API can exist in multiple environments. Additionally, you can create multiple instances of an API in the same environment. 
 
-You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies (for example, to the proxy), and throttling and security policies to the basic endpoint.
+An example: 
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
+* Create an instance of an API and configure it as a proxy. Apply caching policies to this proxy. 
 
-To create a API instance:
+* Then, create an API instance to manage directly as a basic endpoint. Apply throttling and security policies to this basic endpoint. 
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+* After configuring an API version for management, repeat the process to create another API instance. Specifying the same version creates another API instance of the version. 
+
+* Finally, create multiple API instances and click the version information to view API details.
+
+The following procedure describes creating an API instance from Exchange, and configuring the API instance as an endpoint with proxy.
```

Comment:
Move this to follow the next subtitle and introduce the task therein.

We avoid wording that creates an organizational interdependency among topics ("the following..." with reference to a separate topic).

Reply:
Fixed.